src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlSerialMethodWriter.java

```diff
@@ -73,9 +71,7 @@ public Content getSerializableMethodsHeader() {
      */
     @Override
     public Content getMethodsContentHeader(boolean isLastContent) {
-        HtmlTree li = new HtmlTree(TagName.LI);
-        li.setStyle(HtmlStyle.blockList);
-        return li;
+        return new HtmlTree(TagName.LI).setStyle(HtmlStyle.blockList);
```

Comment:
While I agree that this is a direct simplification of the existing code, I thought our CSS style was to put/declare the style just on the `ul` element, and then use `ul.STYLE > li` in the stylesheet.

Reply:
You are right, the class attribute is not needed. I left it there because my goal was not to cause any output changes. However, after removing the attribute, tests still pass and the change in the "Serialized Form" page does not affect the appearance of the page (I checked both in the browser and in the stylesheet). I'll therefore add a commit to remove the style from this `<li>` element.